src/hooks/useCustomBlur.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback, useEffect } from 'react';
+
+const useCustomBlur = (isOpen: boolean, closeMenu: () => void) => {
+  const handleDocumentClick = useCallback(
+    (event: MouseEvent) => {
+      event.preventDefault();
+      event.stopPropagation();
```

Comment:
This would potentially break any click events that are attached to the window 🤔 

Not sure what could possibly be affected by this but wonder if there is a way where we don't have to use preventDefault/stopPropagation to prevent any unintended behavior changes.

Reply:
Great callout